packages/react/src/components/Text/Text.tsx

```diff
@@ -103,8 +103,6 @@ class Text extends UIComponent<WithAsProp<TextProps>, any> {
 Text.create = createShorthandFactory({ Component: Text, mappedProp: 'content' })
 
 /**
- * A Text component formats occurrences of text consistently.
- * @accessibility
- * Text is how people read the content on your website.
+ * A Text consistently styles and formats occurrences of text.
  */
```

Comment:
Why would I used Text component, not just React <span> for example?

Reply:
because it would inconsistently style and format text for you? :) could you, please, suggest what you think should be added here?